packages/vite/src/vite-node.ts

```
@@ -1,22 +1,129 @@
+import type { FetchResult, ViteNodeResolveId } from 'vite-node'
+import { provider } from 'std-env'
+
 import { distDir } from './dirs'
 import type { ViteBuildContext } from './vite'
 import { isCSS } from './utils'
 
-// TODO: Remove this in favor of registerViteNodeMiddleware
-// after Nitropack or h3 allows adding middleware after setup
+export interface ViteNodeRequestMap {
+  manifest: {
+    request: void
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)